src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -316,7 +316,7 @@ They can be configured through the settings file 'winget settings'.</value>
     <comment>Links to different webpages</comment>
   </data>
   <data name="ListCommandLongDescription" xml:space="preserve">
-    <value>The list command displays the packages installed on the system, as well as whether an update is available. Additional options can be provided to filter the output, much like the search command.</value>
+    <value>The list command displays the packages installed on the system, as well as whether an upgrade is available. Additional options can be provided to filter the output, much like the search command.</value>

```

Comment:
Update `install` command description to include the new behavior of changing to `upgrade`?